src/arcade/games/tictactoe/tictactoe.py

```diff
@@ -0,0 +1,220 @@
+import pygame as pg
+import math
+from arcade import plethoraAPI
+
+class Board(plethoraAPI.Game):
+    def __init__(self):
+        super().__init__(size=(800,800), fps=20)
+        self.moveCount=0;
+        self.box=-1
+        self.turn = 'X'
+        self.moves=['-'] *9              #array to track spaces
+        self.win='-'
+        self.inendgame = False
+
+        self.clock=pg.time.Clock()
+        self.dim=800                    #size of screen
+        self.sqDim= 200
+        self.square=(self.sqDim,self.sqDim)
+    #    self.size= (self.dim, self.dim) #dimension for entire screen
+        self.pad=100                    #all sides have 100 buffer
+        self.center=(self.dim//2, self.pad//2)
+
+        self.white= (255,255,255)
+        self.black= (0,0,0)
+        self.blue= (0,0,128)
+        self.red=(255,0,0)
+        self.gray=(80,80,80)
+
+        self.font = pg.font.Font('freesansbold.ttf', 16)
+        self.message="Welcome to Tic-Tac-Toe, X goes first. Select a space"
+        self.text = self.font.render(self.message, True, self.black, self.white)
+        # pg.display.set_caption("Tic-Tac-Toe")
+        self.rects = [
+        pg.Rect((self.pad,self.pad),(self.square)),                  #1 2 3
+        pg.Rect((self.pad+self.sqDim,self.pad),(self.square)),
+        pg.Rect((self.pad+self.sqDim*2,self.pad),(self.square)),
+
+        pg.Rect((self.pad,self.pad+self.sqDim),(self.square)),      #4 5 6
+        pg.Rect((self.pad+self.sqDim,self.pad+self.sqDim),(self.square)),
+        pg.Rect((self.pad+self.sqDim*2,self.pad+self.sqDim),(self.square)),
+
+        pg.Rect((self.pad,self.pad+self.sqDim*2),(self.square)),       #7 8 9
+        pg.Rect((self.pad+self.sqDim,self.pad+self.sqDim*2),(self.square)),
+        pg.Rect((self.pad+self.sqDim*2,self.pad+self.sqDim*2),(self.square))
+        ]
+
+    def onrender(self):
```

Comment:
I think this is why your end game window never shows up. ‘onrender()’ is called after PlethoraPy has rendered, and does not run again unless ‘onevent()’ returns True or ‘onrender()’ returns True. If you setup the endgame window within onrender and then return False, your window won’t ever update to show the endgame graphics.

Reply:
You were right. I found the minor issues and corrected them